sra_experiment/1_download_process_data.ipynb

```diff
@@ -0,0 +1,12781 @@
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.300] [puff::index::jointLog] [info] Total # of Contigs : 7,451\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.300] [puff::index::jointLog] [info] Total # of numerical Contigs : 7,451\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.300] [puff::index::jointLog] [info] Total # of contig vec entries: 8,421\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.300] [puff::index::jointLog] [info] bits per offset entry 14\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.300] [puff::index::jointLog] [info] Done constructing the contig vector. 7452\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.301] [puff::index::jointLog] [info] # segments = 7,451\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.301] [puff::index::jointLog] [info] total length = 5,641,451\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.302] [puff::index::jointLog] [info] Reading the reference files ...\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.314] [puff::index::jointLog] [info] positional integer width = 23\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.314] [puff::index::jointLog] [info] seqSize = 5,641,451\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.314] [puff::index::jointLog] [info] rankSize = 5,641,451\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.314] [puff::index::jointLog] [info] edgeVecSize = 0\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.314] [puff::index::jointLog] [info] num keys = 5,417,921\n",
+      "\u001b[00mfor info, total work write each  : 2.331    total work inram from level 3 : 4.322  total work raw : 25.000 \n",
+      "[Building BooPHF]  100  %   elapsed:   0 min 0  sec   remaining:   0 min 0  sec\n",
+      "Bitarray        28393984  bits (100.00 %)   (array + ranks )\n",
+      "final hash             0  bits (0.00 %) (nb in final hash 0)\n",
+      "\u001b[00m[2020-07-08 15:49:21.503] [puff::index::jointLog] [info] mphf size = 3.38483 MB\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.509] [puff::index::jointLog] [info] chunk size = 2,820,726\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.509] [puff::index::jointLog] [info] chunk 0 = [0, 2,820,726)\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.509] [puff::index::jointLog] [info] chunk 1 = [2,820,726, 5,641,421)\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.816] [puff::index::jointLog] [info] finished populating pos vector\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.816] [puff::index::jointLog] [info] writing index components\n",
+      "\u001b[00m\u001b[00m[2020-07-08 15:49:21.828] [puff::index::jointLog] [info] finished writing dense pufferfish index\n",
+      "\u001b[00m[2020-07-08 15:49:21.829] [jLog] [info] done building index\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Get PAO1 index\n",
+    "! salmon index -t ~/Documents/Data/Core_accessory/Pseudomonas_aeruginosa_PAO1_107.ffn.gz -i ~/Documents/Data/Core_accessory/pao1_index"
```

Comment:
This is sooo much output. I'm not a jupyter expert, but is there a way to write the stdout to a txt file instead of having it be written into the document? I imagine you won't need to reference it as much as the notebook itself, and that would save you like 10,000 lines.

Reply:
I very much agree. Looks like its a feature for downstream analysis to print to stdout: https://github.com/COMBINE-lab/salmon/issues/242